### ISSUE TITLE
Text Encoding Issues

### ISSUE THREAD

**Issue by typkrft** [hay-kot/mealie#90]
There seem to be some symbols not getting encoded correctly. 

From `https://ohsweetbasil.com/instant-pot-award-winning-chili-recipe/`

In Step 8:
```
Turn the instant pot to chili (if you don&#x27;t have a &quot;chili&quot; setting, use the &quot;manual&quot; setting) and cook for 18-20 minutes*. Allow pressure to release for 10-15 minutes or quick release with the vent.
```

**Comment by goose-ws** [comment]
+1 rather than opening a new issue, it looks like the encoder for HTML characters isn't parsing in some places, particularly description and ingredients.

Sample URL: https://www.purewow.com/recipes/Slow-Cooker-Pulled-Chicken

![image](https://user-images.githubusercontent.com/874014/104982569-f1d4df80-59d8-11eb-8041-607739d4ae89.png)

**Comment by hay-kot** [comment]
Fixed in the [new release v0.2.0](https://github.com/hay-kot/mealie/releases/tag/v0.2.0) 🎉
